packages/ui/src/components/MetadataEditor/PipeErrorHandlerEditor.tsx

```diff
@@ -58,16 +58,19 @@ export const PipeErrorHandlerEditor: FunctionComponent<PropsWithChildren<PipeErr
     <>
       <Select
         id={'pipe-error-handler-select'}
+        data-testid={'pipe-error-handler-select'}
         isOpen={isOpen}
         selected={errorHandlerType}
         onSelect={onSelect}
         toggle={toggle}
       >
-        <SelectList>
+        <SelectList data-testid={'pipe-error-handler-select-list'}>
```

Comment:
This can also be written as 
```suggestion
        <SelectList data-testid="pipe-error-handler-select-list">
```